R/a11y_panel.R

```diff
@@ -0,0 +1,250 @@
+#' Accessibility panel
+#'
```

Comment:
Worth adding some notes about how this function should only be used in conjunction with a review from our team? I wouldn't really want anyone running off and making their own statements that look correct, but might be misguided or missing key things?

Reply:
Could also be worth pointing to the GDS model accessibility statement if anyone wants to read more into it (aware that's a niche audience, though better to set the expectation that some people should be interested in that!) - https://www.gov.uk/guidance/model-accessibility-statement

Reply:
Done